src/executor/mod.rs

```diff
@@ -44,79 +44,10 @@
 pub mod current_thread;
 
 #[deprecated(since = "0.1.8", note = "use tokio-threadpool crate instead")]
-#[doc(hidden)]
+/// Re-exports of [`tokio-threadpool`], deprecated in favor of the crate.
```

Comment:
Could you clarify why you removed `#[doc(hidden)]`?

Reply:
I removed `#[doc(hidden)]` so that the deprecation tag plus the short note and link I added to tokio-threadpool is preserved.  As of the [last 0.1.7 release](https://docs.rs/tokio/0.1.7/tokio/executor/thread_pool/index.html) there was the full documents here. Otherwise, on current master (without my change) the module just vanishes from the rustdoc, without even the deprecation tag being visible, which isn't super helpful to anyone currently using that path and wondering how to upgrade.

Reply:
The goal is the deprecation message would provide the upgrade hint. The pattern in the past is to hide deprecated APIs from docs.

Reply:
Irrespective of conventions, my change improves the rustdoc by showing the deprecation and a link.  The deprecation note is still useful/visible when compiling something that uses this path.   Also I removed the much larger doc-comment section that would show up without the `doc(hidden)`, so that is also at parity. 

If you feel strongly that deprecated re-exports must not be documented (I respectfully disagree), then I can put the `#[doc(hidden)]` back and remove my new doc-comment and link.

Reply:
Either way, it can be sorted later.